### ISSUE TITLE
crypto/x509: TestSystemVerify consistently failing

### ISSUE THREAD

**Issue by bcmills** [golang/go#58791]
```
#!watchflakes
post <- goos == "windows" && pkg == "crypto/x509" && test == "TestSystemVerify"
```

https://build.golang.org/log/97b5cac923dc9079392efd9637b9fffa8b9f7592:
```
--- FAIL: TestSystemVerify (1.59s)
    --- FAIL: TestSystemVerify/Valid (0.10s)
        verify_test.go:479: unexpected error: x509: certificate signed by unknown authority
    --- FAIL: TestSystemVerify/MixedCase (0.00s)
        verify_test.go:479: unexpected error: x509: certificate signed by unknown authority
    --- FAIL: TestSystemVerify/HostnameMismatch (0.00s)
        verify_test.go:378: error was not a HostnameError: x509: certificate signed by unknown authority
    --- FAIL: TestSystemVerify/IPMissing (0.00s)
        verify_test.go:378: error was not a HostnameError: x509: certificate signed by unknown authority
    --- FAIL: TestSystemVerify/Expired (0.00s)
        verify_test.go:388: error was not Expired: x509: certificate signed by unknown authority
    --- FAIL: TestSystemVerify/RootInIntermediates (0.00s)
        verify_test.go:479: unexpected error: x509: certificate signed by unknown authority
    --- FAIL: TestSystemVerify/SHA-384 (0.02s)
        verify_test.go:514: Got 0 matches instead of 1 for expected chain [api.moip.com.br COMODO RSA Extended Validation Secure Server CA COMODO RSA Certification Authority AddTrust External CA Root]
FAIL
FAIL	crypto/x509	2.845s
```

(attn @golang/security @dmitshur)

**Comment by bcmills** [comment]
I think this is something to do with an expiration — all of the builders started failing at once, even on TryBots at older base CLs (https://storage.googleapis.com/go-build-log/cd3d6904/windows-386-2016_967987aa.log).

**Comment by dmitshur** [comment]
@gopherbot Please backport at minimum the Windows text fix aspect of this issue.

This is needed to fix failing tests on release branches.

**Comment by dmitshur** [comment]
Thanks watchflakes, but commit 600d37 is before the fix in commit [21e451e](https://go.googlesource.com/go/+/21e451ec446724c9f3f124139e07d88ff812a1bc).

**Comment by rolandshoemaker** [comment]
These all appear old.
